packages/build-tools/src/utils/expoUpdates.ts

```diff
@@ -19,10 +21,26 @@ import { BuildContext } from '../context';
 
 import isExpoUpdatesInstalledAsync from './isExpoUpdatesInstalled';
 
+export const setRuntimeVersionNativelyAsync = async (
+  ctx: BuildContext<Job>,
+  runtimeVersion: string
+): Promise<void> => {
```

Comment:
Is there a particular reason why functions in this file are not just functions but functions assigned to variables?
If not, can you refactor the file?